nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -2,6 +2,7 @@ define([], () => {
     'use strict';
 
     const appStates = [
+        // new
```

Comment:
added in these comments to make it easier to see what was going on when all the nodes are collapsed!